config/initializers/rack_attack.rb

```diff
@@ -18,7 +18,9 @@ def remote_ip
     # whitelisting). It must implement .increment and .write like
     # ActiveSupport::Cache::Store
 
-    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(Redis.current)
+    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(
+      Redis.new(url: Figaro.env.sidekiq_redis_url)
```

Comment:
If we want to share the redis instance with rack attack and sidekiq we should call the variable something else.  Maybe just `redis_url`?